src/NUnitFramework/framework/Internal/Builders/DefaultSuiteBuilder.cs

```diff
@@ -176,7 +176,7 @@ private bool HasArguments(IFixtureBuilder attr)
             // Only TestFixtureAttribute can be used without arguments
             var temp = attr as TestFixtureAttribute;
 
-            return temp == null || temp.Arguments.Length > 0 || temp.TypeArgs.Length > 0;
+            return temp == null || (temp.Arguments != null && temp.Arguments.Length > 0) || (temp.TypeArgs != null && temp.TypeArgs.Length > 0);
         }
```

Comment:
The intent of the code is that `TestFixture(null)` should be legal and equivalent to `TestFixture(new object[] { null })]`. I'm pretty sure that once worked, but perhaps it isn't protected by a test and something changed.

Probably, we need to review all the alternative constructors of `TestFixtureAttribute` and simplify.